src/runtime/arc.ts

```diff
@@ -264,7 +264,7 @@ constructor({id, context, pecFactories, slotComposer, loader, storageKey, storag
     switch (key.protocol) {
       case 'firebase':
       case 'pouchdb':
-        context.handles += `store ${id} of ${store.type.toString()} ${combinedId} @${store.version === null ? 0 : store.version} ${handleTags} at '${store.storageKey}'\n`;
+        context.handles += context.handles += store.toManifestString({handleTags, overrides: {name}}) + '\n';
```

Comment:
toManifestString should probably add the `#` characters, rather than expecting tags with `#` already added.

Reply:
Done